Nbs/Projects/Domain/Sources/SwiftDataModel/Category.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  Category.swift
+//  Domain
+//
+//  Created by 여성일 on 10/17/25.
+//
+
+import Foundation
+import SwiftData
+
+@Model
+public final class Category {
+  @Attribute(.unique) public var categoryName: String // 카테고리 이름
+  public var createdAt: Date
+  
+  @Relationship(inverse: \LinkItem.category) public var links: [LinkItem] = []
+  
+  public init(categoryName: String) {
+    self.categoryName = categoryName
+    self.createdAt = Date()
```

Comment:
```suggestion
    self.createdAt = .now
```

생성자에서 만들어진 시점이 now로 되게 설정하는건 어떠 신가염

Reply:
긍검